### PR TITLE
MAILET-156 AutomaticallySentMailDetectorImpl should not rely on Mime4…

### PR DESCRIPTION
**Pull request by chibenwa**
…J default configuration

### CODE REVIEW

File: mailet/base/src/main/java/org/apache/mailet/base/AutomaticallySentMailDetectorImpl.java [hunk 1]
```diff
@@ -36,6 +36,14 @@
 
 public class AutomaticallySentMailDetectorImpl implements AutomaticallySentMailDetector {
 
+    private static final MimeConfig MIME_ENTITY_CONFIG = MimeConfig.custom()
+        .setMaxContentLen(-1)
+        .setMaxHeaderCount(-1)
+        .setMaxHeaderLen(-1)
+        .setMaxHeaderCount(-1)
```
Comment (quynhn) [inline mailet/base/src/main/java/org/apache/mailet/base/AutomaticallySentMailDetectorImpl.java]: duplicated line "setMaxHeaderCount(-1)"

File: mailet/base/src/main/java/org/apache/mailet/base/AutomaticallySentMailDetectorImpl.java [hunk 2]
```diff
@@ -41,7 +41,6 @@
         .setMaxHeaderCount(-1)
         .setMaxHeaderLen(-1)
         .setMaxHeaderCount(-1)
-        .setMaxLineLen(-1)
```
Comment (rouazana) [inline mailet/base/src/main/java/org/apache/mailet/base/AutomaticallySentMailDetectorImpl.java]: wow, you did not remove the good line, please be more careful

File: mailet/base/src/test/java/org/apache/mailet/base/AutomaticallySentMailDetectorImplTest.java [hunk 3]
```diff
@@ -286,4 +288,24 @@ public void isMdnSentAutomaticallyShouldManageItsMimeType() throws Exception {
...
+                .data("The body part have 1010 headers, which overpass MIME4J default limits")
+                .build())
+            .build();
+
+        FakeMail fakeMail = FakeMail.builder()
+            .sender(MailAddressFixture.ANY_AT_JAMES)
+            .mimeMessage(message)
+            .build();
+
+        assertThat(new AutomaticallySentMailDetectorImpl().isMdnSentAutomatically(fakeMail)).isFalse();
```
Comment (strokyl) [inline mailet/base/src/test/java/org/apache/mailet/base/AutomaticallySentMailDetectorImplTest.java]: Stupid question what does `Mdn` in `isMdnSentAutomatically`

Comment (chibenwa) [inline mailet/base/src/test/java/org/apache/mailet/base/AutomaticallySentMailDetectorImplTest.java]: It stands for https://tools.ietf.org/html/rfc3798

Message Disposition Notification

**Comment by chibenwa** [comment]
At this point PR build was green

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**